spec.md

```diff
@@ -2295,34 +2350,12 @@ be inferred.
 
 ``` firrtl
 SInt(-42)
+SInt("b-101010")
```

Comment:
is there supposed to be some words above that say something about the width of these, or what these are all showing? It seems like these are presented without explanation, unlike the integer version of these which states that the bit width will be 5 for all of them